scylla_operations/Cargo.toml

```diff
@@ -13,7 +13,7 @@ tokio = {version = "1.20", features = ["macros", "rt"] }
 chrono = { version = "0.4.19", features = ["serde"] }
 # Logging
 log = { version = "0.4", features = ["kv_unstable", "std", "kv_unstable_serde"] }
-env_logger = "0.9"
+env_logger = "0.11.2"
```

Comment:
Shouldn't we change the version in `cargo.toml` from old version to `0.1.23` for all the packages?

Reply:
Not all the packages are updates as a part of this change. That is the reason I left the ones with no cargo.toml version bump. Thats the same reason why CI workflow is failing as its trying to push to npm with the same version. It doesnt seem correct to release all the artefacts in case one  has a change.

Reply:
Although this is on way, then you will have to change the npm publish action/script to publish only the ones that is changed. 

In Talos, we went with the strategy to bump the version of all packages, even if they are not updated, so that the version number is in sync across packages. I think this would be easier with respect to maintainability